djaoapp/static/scss/base/djaodjin-dashboard.scss

```diff
@@ -587,4 +587,44 @@ $dashboard-icon-color-active:   $dashboard-sidebar-bg;
 
 .icon-active {
     background: $dashboard-icon-color-active;
-}
\ No newline at end of file
+}
+
+.chart-area {
+    height: 400px !important;
+    width: 100%;
+}
+
+.graph-legend-dot {
+    height: 12px;
+    width: 12px;
+    background: #ccc;
+    border-radius: 50%;
+    display: inline-block;
+}
+
+.nv-legendWrap {
```

Comment:
There has to be another way to hide the legend in the chart. Setting `.nv-legendWrap` globally will introduce a lot of frustrating debugging time when someone is wondering why their legend is not showing up on a totally unrelated chart they are working on.